build.gradle.kts

```
@@ -90,6 +90,8 @@ dependencies {
     //Sets the dependencies for the examples
     configurations.asMap["examplesCompile"] = configurations["apiElements"]
     configurations.asMap["examplesRuntime"] = configurations["implementation"]
+
+    testCompile("org.junit.jupiter:junit-jupiter:5.4.0")
```

```suggestion
    testImplementation("org.junit.jupiter:junit-jupiter:5.4.0")
```
`testCompile` is deprecated: https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management